schema/make_schema.py

```diff
@@ -166,6 +166,9 @@ def format_param(param: click.core.Option) -> SamCliParameterSchema:
             formatted_param_types.append("array")
         else:
             formatted_param_types.append(param_name or "string")
+    formatted_param_types = [
```

Comment:
Discussed offline to use sets and calling sorted here for simplicity and consistency.